crates/core/src/node/inner/node_executor.rs

```diff
@@ -111,7 +112,7 @@ impl NodeExecutor {
         // Reply to sender if we can, otherwise hold result for further processing
         let result = if let Some(reply) = reply {
             if let Err(result) = reply.send(result) {
-                tracing::info!("failed to reply as receiver has been dropped");
+                sh_eprintln!("failed to reply as receiver has been dropped");
```

Comment:
I would leave reply failures as `tracing` logs. If receiver was dropped then it (likely) means network connection with users client (ethers, hardhat script etc) got severed for some reason. Not super actionable from user's side.